src/components/PVText.tsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react'
 import { Text } from 'react-native'
-import { useGlobal } from 'reactn'
```

Comment:
If we don't actually use the global theme in here, do we even need to create a custom component? I added the `style={[props.style, globalTheme.text]}` piece because that way, we can simply override the global style only if we want to. and the rest of the text should be the same so that we can apply the global theme correctly.